tardis/io/util.py

```diff
@@ -288,6 +289,27 @@ def from_hdf_util(cls, file_path, path=''):
                             hdf[key] = None
         return hdf
 
+    @classmethod
+    def from_hdf(cls, file_path, path='', name=None):
+
+        if name is None:
+            name = cls.__name__
+
+        buff_path = os.path.join(path, name)
+        data = cls.from_hdf_util(file_path, buff_path)
+        for key, value in cls.class_properties.items():
+            data[key] = value.from_hdf(file_path, buff_path, name=key)
+
+        #Get initialization paramaters from constructor definition of class
+        argspec = inspect.getargspec(cls.__init__).args
```

Comment:
I like this line a lot. I previously was thinking about removing `hdf_properties` and using the arguments of `__init__`.

Well done!